[2024-05-25T21:56:01.294+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-25T21:56:01.316+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T21:55:00+00:00 [queued]>
[2024-05-25T21:56:01.323+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T21:55:00+00:00 [queued]>
[2024-05-25T21:56:01.324+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-25T21:56:01.338+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): first-step-el> on 2024-05-25 21:55:00+00:00
[2024-05-25T21:56:01.343+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1475) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-25T21:56:01.345+0000] {standard_task_runner.py:63} INFO - Started process 1477 to run task
[2024-05-25T21:56:01.345+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'DAG_catedra_***', 'first-step-el', 'scheduled__2024-05-25T21:55:00+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/dag_clase.py', '--cfg-path', '/tmp/tmpyp5v4bhi']
[2024-05-25T21:56:01.346+0000] {standard_task_runner.py:91} INFO - Job 278: Subtask first-step-el
[2024-05-25T21:56:01.386+0000] {task_command.py:426} INFO - Running <TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T21:55:00+00:00 [running]> on host b2cfee8f0b2b
[2024-05-25T21:56:01.471+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_catedra_***' AIRFLOW_CTX_TASK_ID='first-step-el' AIRFLOW_CTX_EXECUTION_DATE='2024-05-25T21:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-25T21:55:00+00:00'
[2024-05-25T21:56:01.472+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-25T21:56:01.478+0000] {logging_mixin.py:188} INFO - Connection to database was succesful
[2024-05-25T21:56:01.493+0000] {logging_mixin.py:188} INFO - Query completed
[2024-05-25T21:56:01.496+0000] {logging_mixin.py:188} INFO - execute_query_result_set()
[2024-05-25T21:56:01.501+0000] {logging_mixin.py:188} INFO - Connection to database was succesful
[2024-05-25T21:56:01.509+0000] {logging_mixin.py:188} INFO - Inserts performed wihout fail
[2024-05-25T21:56:01.509+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-25T21:56:01.509+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-25T21:56:01.517+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=DAG_catedra_***, task_id=first-step-el, run_id=scheduled__2024-05-25T21:55:00+00:00, execution_date=20240525T215500, start_date=20240525T215601, end_date=20240525T215601
[2024-05-25T21:56:01.558+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-25T21:56:01.593+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-25T21:56:01.599+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
